app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -655,12 +655,6 @@ void OpenXRInputSource::EmulateControllerFromHand(device::RenderMode renderMode,
             delegate.SetSqueezeActionStop(mIndex);
         }
     }
-
-    // Menu button
-    bool ringPinching = GetDistanceBetweenJoints(XR_HAND_JOINT_THUMB_TIP_EXT,
-                                                 XR_HAND_JOINT_RING_TIP_EXT) < kPinchThreshold;
-    delegate.SetButtonState(mIndex, ControllerDelegate::BUTTON_APP, -1, ringPinching,
-                            ringPinching, 1.0);
 }
```

Comment:
We need a back action for hand tracking otherwise you cannot exit WebXR or immersive videos